.github/workflows/portal-loop.yml

```diff
@@ -46,3 +48,57 @@ jobs:
           push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+
+  test-portal-loop-docker-compose:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v4
+
+      - name: "Setup the images"
+        run: |
+          cd misc/loop
+
+          docker compose build
+          docker compose pull
+          docker compose up -d
+
+      - name: "Test1 - Portal loop start gnoland"
+        run: |
+          while
+            block_height=$(curl -s localhost:26657/status | jq -r '.result.sync_info.latest_block_height')
+            echo "Current block height: $block_height"
+            [[ "$block_height" -lt 10 ]]
+          do
+            sleep 1
+          done
+
+          curl -s localhost:26657/status | jq
+
+      - name: "Buid new gnolang/gno image"
+        run: |
+          docker build -t ghcr.io/gnolang/gno -f Dockerfile.gnoland.release .
```

Comment:
I think we swapped the name from `gnolang/gno` to `gnolang/gnoland` for the chain binary

Reply:
`gnolang/gno/gnoland` ... but with the new PR https://github.com/gnolang/gno/pull/2199